readthedocs/doc_builder/templates/doc_builder/conf.py.tmpl

```diff
@@ -50,12 +48,6 @@ if 'html_static_path' in globals():
 else:
     html_static_path = ['_static', '{{ static_path }}']
 
-# Add RTD Theme Path. 
-if 'html_theme_path' in globals():
-    html_theme_path.append('{{ template_path }}')
-else:
-    html_theme_path = ['_themes', '{{ template_path }}']
-
```

Comment:
I think this broke the build for us: https://readthedocs.org/projects/lasagne/builds/
Before this PR it worked, now it says "ThemeError: no theme named 'sphinx_rtd_theme' found (missing theme.conf?)"
The relevant section in our `conf.py` is as follows:
```python
if os.environ.get('READTHEDOCS') != 'True':
    try:
        import sphinx_rtd_theme
    except ImportError:
        pass  # assume we have sphinx >= 1.3
    else:
        html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
html_theme = 'sphinx_rtd_theme'
```
That is, if running outside of readthedocs and with an old version of Sphinx, it imports `sphinx_rtd_theme` and sets `html_theme_path`. Then it sets `html_theme`, no matter if running on readthedocs or not. This worked fine, now it doesn't.

Note that we have Sphinx==1.2.3 in our requirements.txt because of https://github.com/sphinx-doc/sphinx/issues/1822 (which will only be fixed in Sphinx 1.4). We then relied on readthedocs setting the theme path for us.

Why has this been removed, and how can we work around it? (I assume if we just remove the `os.environ.get('READTHEDOCS')` check, we'd fall back to using the PyPI version of sphinx-rtd-theme. But I'd rather like to hear your suggestion than committing a couple of different things to try!)

Thank you!